scripts/globals/spells/esuna.lua

```diff
@@ -84,8 +84,10 @@ spell_object.onSpellCast = function(caster, target, spell)
             end
         end
 
+        local delEff
+
         if (statusNum >= 0) then -- make sure this happens once instead of for every target
-            local delEff = math.random(0, statusNum) -- pick a random status to delete
+            delEff = math.random(0, statusNum) -- pick a random status to delete
```

Comment:
Need to look at some lua scope things, but delEff is used outside of that block several times below.  Might need to have it declared there.